doc/TestingApplications/IntegrationTesting/Overview/pipelineconfiguration.md

```diff
@@ -1,17 +1,98 @@
-### Configure Pipeline job for running Integration test cases
- Once Application deployed and automatically Integration testing performed. A pipeline script written in jenkins environment and contains below stages.
-- source stage
-- Setup to run postman collection file
-- Run the test suites using newman
-- Results available
-- Email Notification
+## Integration Testing automation procedure 
+Once Application deployed and automatically testsuites execution performed on production test server.
+
+As a prerequisite, the related test tools mentioned earlier to be installed and set up should done the server.
 
 ![IntegrationFlow](./../Images/IntegrationFlow.PNG)
```

Comment:
seems we need to beautify the image .. the line seems a bit slanty. Could you please look into this